frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -44,6 +44,10 @@ const shortenImage = (img: string) =>
     .slice(1, 3)
     .join('/');
 const shortenHash = (hash: string) => hash.slice(7, 18);
+const totalCount = (obj: ImageManifestVuln) => {
+  const { highCount, mediumCount, lowCount, unknownCount } = obj.status;
+  return (highCount || 0) + (mediumCount || 0) + (lowCount || 0) + (unknownCount || 0);
```

Comment:
shall we have check for status as well, see if it make sense

```suggestion
  if(!obj.status) return 0;
  const { highCount =0, mediumCount =0, lowCount =0 , unknownCount =0 } = obj.status;
  return highCount + mediumCount + lowCount + unknownCount;
```